src/FSharpVSPowerTools.Logic/Outlining/OutliningTagger.fs

```diff
@@ -235,31 +263,28 @@ type OutliningTagger
     /// viewUpdate -=> doUpdate -=> triggerUpdate -=> tagsChanged -=> getTags
     let getTags (normalizedSnapshotSpans: NormalizedSnapshotSpanCollection) : IOutliningRegionTag ITagSpan seq =
         match Seq.isEmpty normalizedSnapshotSpans, Array.isEmpty scopedSnapSpans with
-        | false, false -> 
+        | false, false ->
```

Comment:
Because it's better than `| true`. 
Yes, it will cause a Choice allocation, but it's all in gen 0, so no problem here.